components/post_view/post_list_normal/post_list_normal.jsx

```diff
@@ -0,0 +1,627 @@
+                    shouldHighlight={this.props.focusedPostId === post.id}
+                />
+            );
+
+            const currentPostDay = Utils.getDateForUnixTicks(post.create_at);
+            if (currentPostDay.toDateString() !== previousPostDay.toDateString()) {
+                postCtls.push(
+                    <DateSeparator
+                        key={currentPostDay}
+                        date={currentPostDay}
+                    />
+                );
+            }
+
+            const isNotCurrentUser = post.user_id !== currentUserId || isFromWebhook(post);
+            if (isNotCurrentUser &&
+                    lastViewed !== 0 &&
+                    post.create_at > lastViewed &&
+                    !Utils.isPostEphemeral(post) &&
+                    !renderedLastViewed) {
+                renderedLastViewed = true;
+
+                // Temporary fix to solve ie11 rendering issue
+                let newSeparatorId = '';
+                if (!UserAgent.isInternetExplorer()) {
+                    newSeparatorId = 'new_message_' + post.id;
+                }
+                postCtls.push(
+                    <div
+                        id={newSeparatorId}
+                        key='unviewed'
+                        ref='newMessageSeparator'
+                        className='new-separator'
+                    >
+                        <hr
+                            className='separator__hr'
+                        />
+                        <div className='separator__text'>
+                            <FormattedMessage
+                                id='posts_view.newMsg'
+                                defaultMessage='New Messages'
+                            />
+                        </div>
+                    </div>
+                );
+            }
+
+            postCtls.push(postCtl);
+            previousPostDay = currentPostDay;
+        }
+
+        return postCtls;
+    }
+
+    render() {
+        const posts = this.props.posts || [];
+        const channel = this.props.channel;
+
+        if ((posts.length === 0 && this.state.isDoingInitialLoad) || channel == null) {
+            return (
+                <div id='post-list'>
+                    <LoadingScreen
+                        position='absolute'
+                        key='loading'
+                    />
+                </div>
+            );
+        }
+
+        let topRow;
+        if (this.state.atEnd) {
+            topRow = (
+                <CreateChannelIntroMessage
+                    channel={channel}
+                />
+            );
+        } else if (this.props.postVisibility >= Constants.MAX_POST_VISIBILITY) {
+            topRow = (
+                <div className='post-list__loading post-list__loading-search'>
+                    <FormattedMessage
+                        id='posts_view.maxLoaded'
+                        defaultMessage='Looking for a specific message? Try searching for it'
```

Comment:
@sudheerDev you need to add this string to the en.json file

Reply:
We could consider just removing this logic. `MAX_POST_VISIBILITY` is 1000000, and there's no way you'll be able to load or render that many posts in the app anyway